shared/reducers/tracker.js

```diff
@@ -604,14 +588,17 @@ function updateProof (proofs: Array<Proof>, rp: RemoteProof, lcr: LinkCheckResul
   return updated
 }
 
-function deriveTrackerState (
-  allOk: boolean,
-  anyWarnings: boolean,
-  anyError: boolean,
-  anyPending: boolean,
-  anyDeletedProofs : boolean,
-  anyUnreachableProofs : boolean,
-  eldestKidChanged: boolean
+export function overviewStateOfProofs (proofs: Array<Proof>): OverviewProofState {
```

Comment:
While I'm in general strongly in favor of splitting this logic up, I think in the future you should have at least 2 uses when you factor code out into a common function. Otherwise it's easy to overfit the original code. Do you have plans to use this function elsewhere?